_posts/2024-02-01-sswg-update-2024.md

```diff
@@ -0,0 +1,87 @@
+
+Since our [previous update post](/blog/sswg-update-2023/), Swift on the Server has continued to mature.
+
+Let's start with a review of the progress made in 2023, before looking at the next steps in 2024.
+
+# 2023 in Review
+
+## Continued focus on growing the ecosystem
+
+The SSWG has continued to focus on growing the ecosystem by incubating packages and providing guidance to package authors. We've seen a lot of new, in development and improved packages this year.
+
+Memcached, which been in development througn GSoC, has made a lot of progress. The wire protocol and higher level connection APIs are done. The remaining work is implementing a connection pool and key sharding. The SSWG is looking forward to continued work on this package.
+
+## Adoption of Structured Concurrency
+
+We've seen a lot of progress on the adoption of Structured Concurrency. Most new libraries are adopting it, and we're working on a guide to help existing libraries adopt it as well. SwiftNIO has been a big driver of this, through the addition of `NIOAsyncChannel`.
+
+Rewrites of ServiceLifecycle, Hummingbird and GRPC are under way that are implemented entirely with Structured Concurrency. Moving to structured concurrency allows NIO to become more of an implementation detail. In addition, structured concurrency provides many advantages to libraries and the adopters, which will be further elaborated on in our guides.
+
+## Tooling Improvements
+
+Tooling has seen some tremendous improvements in 2023. We've seen the introduction of Swiftly, Dependabot for Swift and Chiseled Containers.
+
+In addition, thanks to the introduction of Package Plugins, we've seen the OpenAPI Generator make an entry. This plugin allows you to generate Swift boilerplate code from an OpenAPI specification. This is a great way for building both HTTP clients and servers.
+  
+## Server Community Survey
+
+For the first time, we did run a developer community survey, aimed at collecting feedback and information about the shape and breadth of the Swift Server ecosystem.
+We received a huge amount of responses, for which we'd like to thank everyone! 
+
+We analyzed the responses and used them to how we're doing with current efforts, and help direct the group's goals and efforts in the upcoming year (see below).
+For example, while the majority of respondents already use Swift Concurrency, there remain challanges in some areas with adoption. 
+Among the highlights that shaped our goals for the next year were: a huge interest in standardized tooling for e.g. swift-format, as well as sourcekit-lsp and various IDEs.
+
+We highly encourage you to have a look at the full [SSWG Community Survey 2024 Report](http://TODO), over on the Swift forums!
+
+## New Members
+
+Since our update post of 2023, [Sven A. Schmidt](https://github.com/finestructure) has joined the workgroup. Sven has already been involved with the Documentation WorkGroup, and is known for his work on Swift Package Index.
```

Comment:
```suggestion
Since our update post of 2023, [Sven A. Schmidt](https://github.com/finestructure) has joined the workgroup. Sven has already been involved with the documentation workgroup, and is known for his work on Swift Package Index.
```